pkg/kanctl/actionset_test.go

```diff
@@ -105,3 +105,69 @@ func (k *KanctlTestSuite) TestGenerateActionSetName(c *C) {
 		c.Assert(actual, DeepEquals, tc.expected)
 	}
 }
+
+func (k *KanctlTestSuite) TestParseLabels(c *C) {
+	for _, tc := range []struct {
+		flagValue      string
+		expectedLabels map[string]string
+		expectedErr    error
+	}{
+		{
+			flagValue:      "a=b",
+			expectedLabels: map[string]string{"a": "b"},
+		},
+		{
+			flagValue:      "a=b,c=d",
+			expectedLabels: map[string]string{"a": "b", "c": "d"},
+		},
+		{
+			flagValue:      "a=b,c=d,e=f",
+			expectedLabels: map[string]string{"a": "b", "c": "d", "e": "f"},
+		},
+		{
+			flagValue:      "a=b,c=d,",
+			expectedLabels: nil,
+			expectedErr:    errInvalidFieldLabels,
+		},
+		{
+			flagValue:      ",a=b,c=d,",
+			expectedLabels: nil,
+			expectedErr:    errInvalidFieldLabels,
+		},
+		{
+			flagValue:      ",a=b,c=d",
+			expectedLabels: nil,
+			expectedErr:    errInvalidFieldLabels,
+		},
+		{
+			flagValue:      "a",
+			expectedLabels: nil,
+			expectedErr:    errInvalidFieldLabels,
+		},
```

Comment:
I'm thinking of the following case, should it be considered as an invalid label ? 
```suggestion
		},
		{
			flagValue:      "a=",
			expectedLabels: nil,
			expectedErr:    errInvalidFieldLabels,
		},		
```

Reply:
Hi @e-sumin ,
That's a good point. I just tested labelling a resource with 
```
k label ns default abc=
namespace/default labeled
```
and it worked. So we should support this.

Reply:
In other words, it should be valid label. I have added another test case in our tests.

Reply:
@e-sumin can you please have another look.

Reply:
Since label with key and without label is ok, I'm ok with test you've added.